internal/auth/oidc/ids.go

```diff
@@ -0,0 +1,31 @@
+package oidc
+
+import (
+	"github.com/hashicorp/boundary/internal/db"
+	"github.com/hashicorp/boundary/internal/errors"
+)
+
+const (
+	// AuthMethodPrefix defines the prefix for AuthMethod public ids.
+	AuthMethodPrefix = "amoidc"
+	// AccountPrefix defines the prefix for Account public ids.
+	AccountPrefix = "acctoidc"
```

Comment:
password accounts are `apw`.  This should match by using `aoidc`.

Reply:
I actually debated this and I think a single `a` is a less than ideal prefix for a prefix, so I went with the less ambiguous `acct` for oidc.  We can't change password accounts, but it's not a pattern we have to go forward with.

Reply:
We could also update the password account prefix then and support both prefixes (`apw` and `acctpw`) for the pw accounts.  I think there is benefit in consistency here. @jefferai Do you have thoughts?